vantage6/common/docker_addons.py

```diff
@@ -291,10 +292,27 @@ def remove_container(container: Container, kill=False) -> None:
     if kill:
         try:
             container.kill()
-        except Exception as e:
+        except Exception:
```

Comment:
Maybe container status and check if the container is running first? Then we can catch expections and print them here.

Reply:
Decided not to do this. Checking container status with dockerpy is not as trivial as I'd like it to be. Documentation says status may be 'running', 'exited' or *something else*. As I don't know what the other statuses are, I don't want to be checking on them. 
For now, added a TODO with the suggestion so that we may resolve this in the future. I think it is too low prio to put time into this now